Add blog post for March Standard for Public Code community call (#417)
publiccodenet/blog: _posts/2023-03-06-notes-from-community-call-2 march-2023.md

---
title: "Notes from the Standard for Public Code community call - 6 March 2023"
date: 2023-03-06
author: Jan Ainali
type: blogpost
excerpt: Reflecting on the What this does not do sections
categories:
  - Community call
---

# 2 March 2023 Standard for Public Code community call notes

## Attendees

* [Jan Ainali](https://publiccode.net/who-we-are/team/jan-ainali.html)
* [Eric Herman](https://publiccode.net/who-we-are/team/eric-herman.html)
* Martin Dias d’Ullois
* Johan Groenen

## News from the Foundation for Public Code

* New release for the Governance game, [version 1.0.2](https://github.com/publiccodenet/governance-game/releases/tag/1.0.2)
* We will have a session at MozFest: [In 2030, governments only use open source: a thought experiment](https://schedule.mozillafestival.org/session/BRR3GV-1)

## Notes

> When we recently changed the sections "Why this is important" to instead become an introduction to each criterion, we also noted that many of the sections "What this does not do" did not add a lot of value to the reader, and thus we created the issue [Review "What does this not do" sections #813](https://github.com/publiccodenet/standard/issues/813).

Reviewer: ```suggestion
When we recently changed the "Why this is important" section to instead become an introduction to each criterion, we also noted that many of the "What this does not do"  sections did not add a lot of value to the reader, and so we created the issue [Review "What does this not do" sections #813](https://github.com/publiccodenet/standard/issues/813).
```

Author: Is there a space too many before the newly added "sections"?

> Our first review inspired us to do some addtions to the introductions as some sections had high level insights in them. With that done, we felt that perhaps it would be best to remove all the sections entirely, as the little value that once only was there, have been added to the wider context instead.

Reviewer: ```suggestion
Our first review inspired us to make some additions to the introductions, since some sections had high level insights in them. With that done, we felt it would be best to remove the "What this does not do"  section completely, as the little value that was once only there had been added to the wider context instead.
```

During the call, there were additional insights from Johan and Martin.

> It was observed that since as many of the  "What this does not do" bullet points refer to what other criteria does, those that are not "linking" in such way as to collectively sum up to "What the Standard for Public Code does not do". A unified section with that theme would fit well in the introduction.

Reviewer: ```suggestion
It was observed that since many of the "What this does not do" bullet points refer to other criteria, those that are not linked in such a way collectively sum up "What the Standard for Public Code does not do". A unified section with that theme would fit well in the introduction.
```

It was also noted that unrelated to specific criteria, there are some misconceptions about the Standard for Public Code worth addressing there, for instance, that the Standard for Public Code should have any opinions about specific implementations of a codebase. This section could also be a place for us to differentiate the Standard for Public Code from other standards (we already have [issues on this topic](https://github.com/publiccodenet/standard/issues/200)).

> As we are planning to reframe some of the text in the introduction, to which this suggestion at a glance aligns well with, we will bring this new idea into that work.

Reviewer: ```suggestion
As we are planning to reframe some of the text in the introduction, which this suggestion aligns well with, we will bring this new idea into that work.
```
